oremda/operator.py

```diff
@@ -43,3 +45,33 @@ def execute(self, object_id, params):
     @abstractmethod
     def kernel(self, input_data, parameters):
         pass
+
+
+def operator(func=None, name=None, start=True,
+             plasma_socket_path=DEFAULT_PLASMA_SOCKET_PATH):
+
+    # A decorator to automatically make an Operator where the function
+    # that is decorated will be the kernel function.
+
+    def decorator(func):
+        nonlocal name
+        if name is None:
+            name = func.__name__
+
+        @wraps(func)
+        def kernel(self, *args, **kwargs):
+            # Remove self so the caller does not need to add it
+            return func(*args, **kwargs)
+
+        OpClass = type(name, (Operator,), {'kernel': kernel})
```

Comment:
Just an inconsequential comment, we could have the name of the generated class to be something like `class_name = f"{name.capitalize()}Operator"` to be more in line with python naming conventions

Reply:
Done!